DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: DT
 Type: Package
 Title: A Wrapper of the JavaScript Library 'DataTables'
-Version: 0.4.13
+Version: 0.4.14
```

Comment:
It is fine if you don't bump the version number in each PR. I can do it by myself.